sdcm/utils/version_utils.py

```diff
@@ -69,7 +69,7 @@
 )
 
 SCYLLA_VERSION_RE = re.compile(r"\d+(\.\d+)?\.[\d\w]+([.~][\d\w]+)?")
-SCYLLA_VERSION_GROUPED_RE = re.compile(r'(?P<version>[\w.~]+)-0?\.?(?P<date>[\d]+)\.(?P<commit_id>\w+)')
+SCYLLA_VERSION_GROUPED_RE = re.compile(r'(?P<version>[\w.~]+)-(?P<build>0|rc\d)?\.?(?P<date>[\d]+)\.(?P<commit_id>\w+)')
```

Comment:
would be nice to add a case to unit test for it `test_version_utilis.py`

Reply:
i don't have much resources to do it now... would you mind opening a task for it?